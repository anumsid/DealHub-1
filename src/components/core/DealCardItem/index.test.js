import React from 'react'
import { shallow } from 'enzyme'
import { stub } from 'sinon'
import toJson from 'enzyme-to-json'
import { DealCardItem } from './index'

describe('Deal Card Item Component', () => {
  let wrapper
  beforeEach(() => {
    const props = {
      card: {
        id: 0,
        title: 'Cool Deal',
        votes: ['', ''],
        comments: ['', ''],
        image: '',
        description: 'Hello World',
        location: 'New York',
        category: 'Food & Beverages',
        expiration_date: '2018-07-31T00:00:00.000Z'
      }
    }
    wrapper = shallow(<DealCardItem {...props} />)
  })

  it('Matches snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
