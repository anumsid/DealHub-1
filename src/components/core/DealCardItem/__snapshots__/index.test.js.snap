// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deal Card Item Component Matches snapshot 1`] = `
<Fragment>
  <img
    alt="Card cap"
    className="card-img-top"
    src=""
  />
  <div
    className="card-body"
  >
    <h4
      className="card-title"
    >
      Cool Deal
    </h4>
    <h5
      className="card-title"
    >
      @: 
      New York
    </h5>
    <h5
      className="card-title"
    >
      Food & Beverages
    </h5>
    <h5
      className="card-expiration"
    >
      Until: 
      2018-07-31T00:00:00.000Z
    </h5>
    <p
      className="card-text"
    >
      Hello World
    </p>
  </div>
</Fragment>
`;
